support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -417,9 +417,12 @@ AccessControlled accessControlled() {
                         cookie = UUID.randomUUID().toString();
                         // Switches the label to a self-label, so if the executable is killed and restarted via ExecutorPickle, it will run on the same node:
                         label = computer.getName();
+
```

Comment:
The new whitespace line _here_ is fine, since it directly relates to the code changes being made. Just please avoid changes to _unrelated_ lines, so that diffs are minimal.